src/Tabs/index.jsx

```diff
@@ -113,6 +131,7 @@ Tabs.defaultProps =
     className         : undefined,
     cssMap            : styles,
     onClickTab        : undefined,
+    padding           : 'M',
```

Comment:
For backwards compatibility the default value for `padding` should be `[ 'none', 'M' ]`